### ISSUE TITLE
Accuracy Not improving

### ISSUE THREAD

**Issue by nikhiljaiswal** [lixuan0023/SVHN_RNN#1]
My accuracy is constant to around 0.37 since last many steps. Currently I am on step 130000. How long should I train? This accuracy is constant since step 5000.
![screenshot 496](https://user-images.githubusercontent.com/21051117/34548438-2463d50e-f128-11e7-9ba4-ea2f5d07290c.png)

**Comment by lixuan0023** [comment]
I trained this model for about a week. The training step will be over when `patience = 0`. This model is just for studying Tensorflow, so the performance may not be good.

**Comment by nikhiljaiswal** [comment]
So, if I increase the patience value, and train for a longer time, will I get some improvement? The accuracy is constant to around 0.37 since last 200k epochs. When I see at the accuracy graph provided by you, the accuracy has crossed .80 in first 100k epochs. What could be the reason?

**Comment by lixuan0023** [comment]
![image](https://user-images.githubusercontent.com/25735744/34594268-fd2bceea-f20a-11e7-976d-2b0c2baa6177.png)

Do you download all data? Like the picture below?

**Comment by nikhiljaiswal** [comment]
Yes I have similar structure.

**Comment by lixuan0023** [comment]
Maybe you can adjust `Hyper-parameters`, e.g. leaning rate, decay steps, etc.

**Comment by nikhiljaiswal** [comment]
ok i will try tuning those.

**Comment by nikhiljaiswal** [comment]
Respected Sir,

I have trained the model as per your code. I have a doubt. The accuracy
that you have reported is on the validation set and not on the test set.
How can we measure the test set accuracy?

Anticipating yours response.

Thanking you.

Regards,
Nikhil
M.Tech Scholar
